### ISSUE TITLE
Items duped into PRO-DeathChest on player death

### ISSUE THREAD

**Issue by datig** [RockinChaos/ItemJoin#359]
## Thanks for taking the time to open a bug report!

**Describe the bug**
I installed Pro Death Chest.  Even though I have death-drops enabled the item appears in my chest on death.

Pro Death Chest:
https://www.spigotmc.org/resources/%E2%AD%90pro%E2%AD%90-deathchest-1-8-1-16-x-no-more-losing-items-after-death.63316/

You may be able to duplicate this with another graves plugin.   I only tested with Pro Death Chest.

Here's my item definition in case I set it up incorrectly.

    id: NETHER_STAR
    slot: 8
    name: '&eHub Menu'
    lore:
    - '&7Hub Menu'
    commands:
     - 'myhub'
    itemflags: item-store, death-drops, self-drops, inventory-modify, give-next
    triggers: join, respawn, world-switch
    permission-node: groups.default
    enabled-worlds: ALL

**To Reproduce**
1. Setup item as defined above
2. Install Pro Death Chest
3. Die

**Expected behavior**
I expect my ItemJoin item to not appear in the Death Chest.

**Screenshots**

**Pastebin**
Even if it is not related to the problem, please include ALL configuration files in separate Pastebin.com links as well as any related error codes from the console window.

**Please complete the following information:**
 - Resource Version and Build Number: ItemJoin-v5.1.0
 - Server Version {"oldVersion":"git-Paper-446 (MC: 1.16.5)","currentVersion":"git-Paper-449 (MC: 1.16.5)"}%

**Additional context**

**Comment by RockinChaos** [comment]
So I am currently unable to replicate this with DeathChest v3.4.13 and using the latest ItemJoin snapshot. It seems to function properly (as-in not dropping the item into the chest).

I would suggest updating and let me know if the issue continues;

Link: https://ci.craftationgaming.com/job/ItemJoin/662/
**Remember to remove your old ItemJoin.jar file**

See devlog for changes; https://github.com/RockinChaos/ItemJoin/wiki/DevLog

Do note; This build contains a major change to item commands so any prior commands will break until fixed.
I have removed the `commands:` identifier node from the items. So commands which resembled;
```
    commands:
      interact-right:
      - 'player: deluxemenus open navigation4'
```

Should now look like;
```
    interact-right:
    - 'player: deluxemenus open navigation4'
```

Do make sure you implement proper yaml formatting as when you remove the `commands` node there is two spaces in-front of the interact-right node you need to remove.

**Comment by RockinChaos** [comment]
This bug has been officially fixed in the release of ItemJoin v5.2.0.
You can grab it here; https://www.spigotmc.org/resources/itemjoin.12661/download?version=383801

Please read the changelog as the documentation has not been updated yet;
https://github.com/RockinChaos/ItemJoin/wiki/Recent-Changes****
